lib/shrinkwrap.js

```diff
@@ -110,11 +110,13 @@ function shrinkwrapDeps (deps, top, tree, seen) {
     var childIsOnlyDev = isOnlyDev(child)
     var pkginfo = deps[moduleName(child)] = {}
     var requested = getRequested(child) || child.package._requested || {}
+    var linked = child.isLink || child.isInLInk
```

Comment:
> `var linked = child.isLink || child.isInLInk`

`child.isInLink`?

Reply:
`child.isLink` = "This module is a symlink in your node_modules

`child.isInLink` = "Some parent module of this module is a symlink"

Reply:
There seems to be a typo with a capital i in Link in isInLink.

Reply:
I think it was @christoffer-dropbox who caught it, I just wanted to clarify what I thought he meant.